frontend/hooks/useLocalStorage.js

```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react'
+
+const useLocalStorage = (key, initialValue) => {
+   const [value, setValue] = useState(() => {
+      if (localStorage.getItem(key)) {
```

Comment:
This works great! Another way of doing this would be: 
`const item = localStorage.getItem(key)
  return item ? JSON.parse(item) : initialValue;
`